Makefile

```diff
@@ -177,186 +184,214 @@ function cleanup { \
 }
 endef
 
-# Build manager binary
+##@ Build
+
 .PHONY: manager
-manager: manifests generate fmt vet
+manager: manifests generate fmt vet ## Build the manager binary.
 	go build -o bin/manager internal/main.go
 
-# Run against the configured Kubernetes cluster in ~/.kube/config
 .PHONY: run
-run: manifests generate fmt vet
+run: manifests generate fmt vet ## Run against the configured Kubernetes cluster in ~/.kube/config
```

Comment:
Would this also work if `KUBECONFIG` is set? Maybe like `## Run the controller from your host.`?

Reply:
"Run the controller locally against the cluster configured in $KUBECONFIG" ?

Reply:
Hmm, the variable might not be set tho.

https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#the-kubeconfig-environment-variable

Reply:
I pick the above suggestion from: https://github.com/operator-framework/operator-sdk/blob/e67da35ef4fff3e471a208904b2a142b27ae32b1/testdata/go/v3/memcached-operator/Makefile#L122

Reply:
Actually, the operator-sdk is a little inconsistent about these:

https://github.com/operator-framework/operator-sdk/blob/e67da35ef4fff3e471a208904b2a142b27ae32b1/testdata/go/v3/memcached-operator/Makefile#L160

So, I think this is good for now.

Reply:
Maybe change remove the word `Kubernetes`.